about/index.html

```diff
@@ -96,6 +96,9 @@ <h5 class="subheader">{{officer.title}} ({{officer.year}} )</h5>
 			<li>Ben Bland: Director (2013-2014)</li>
 			<li>Rachel Gregg: Community NEM (2014-2015)</li>
 			<li>Matt Hazley: Projects NEM (2015)</li>
+			<li>Ben Bland: Development NEM (2014-2016)</li>
+      <li>Pip Shea: Director (2015-2016)</li>
```

Comment:
Spacing is pretty horrible as it stands, but keeping it consistent would be good.